### PR TITLE
Blog App: Add forms

### PR DESCRIPTION
**Pull request by BB-Simon**
**The Blog app** will be a classic example of a blog website which will be a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.

### In this PR:
- [x] Create a method called current_user in your ApplicationController to make current user data available to all controllers.
    - It will return the first user from the database.
- [x] Create forms to perform the following functions:
    - Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- [x] Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
```
Comment (Ismailco) [inline app/controllers/comments_controller.rb:1]: - [x] Good job creating the `create` action for the Comments, however you should also create a `new` action in your controllers (comments)

File: app/views/shared/_comment_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<%= form_with model:([post, comment]), html: { class: 'comment-form'} do |form|%>
+  <h1>Add a comment</h1>
+  <%=form.label "Enter Comment", class: 'comment-field-label'%>
+  <%=form.text_area :text, class: 'comment-field'%>
+  <%=form.submit "Comment", class: 'common-btn'%>
+<%end%>
```
Comment (Ismailco) [inline app/views/shared/_comment_form.html.erb]: - [x] Kindly make sure to create a new file `new.html.erb` in the `views/comments/` folder, you can have the choice of referring to the `_comment_form.html.erb` partial there or create a new form inside the `new.html.erb`

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @BB-Simon, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Nice UI
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @BB-Simon 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
Nice design :+1: 
Created view for comments :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**